prime/src/components/borrow/ManageAccountTransactionButton.tsx

```diff
@@ -215,6 +225,44 @@ export function ManageAccountTransactionButton(props: ManageAccountTransactionBu
     accountEtherBalance.value.toNumber() < ANTE &&
     (accountState.liabilities.amount0 > 0 || accountState.liabilities.amount1 > 0);
 
+  const isRemovingToken0Collateral = actionIds.some((id, idx) => {
+    if (id === 1) {
+      const actionArg: string = actionArgs[idx];
+      if (!actionArg) return false;
+      const firstArg: string = `0x${actionArg.slice(26, 66).toLowerCase()}`;
+      return firstArg === token0.address;
+    }
+    return false;
+  });
+  const isSwappingToken0ForToken1 = actionIds.some((id, idx) => {
+    if (id === 6) {
+      const actionArg: string = actionArgs[idx];
+      if (!actionArg) return false;
+      const firstArg: string = `0x${actionArg.slice(26, 66).toLowerCase()}`;
+      return firstArg === token0.address;
+    }
+    return false;
+  });
```

Comment:
In theory you could combine the two `some` predicates and call the resulting boolean `shouldEnableTransfersForToken0`, but I think the verbosity is fine here.

Reply:
I saw this as well but felt being explicit was good; that way, it is clear what each piece is doing and what it means. I was thinking about abstracting the logic more but felt it wasn't worth it now.